app/views/admin/edition_images/_image_upload.html.erb

```diff
@@ -10,7 +10,7 @@
   <%= render "govuk_publishing_components/components/file_upload", {
     name: "image[image_data][file]",
     id: "edition_images_image_data_file",
-    hint: "Images can be JPEG, PNG, SVG or GIF files.",
+    hint: raw("Images can be JPEG, PNG, SVG or GIF files. If you are uploading more than one image, <a class='govuk-link' href='https://www.gov.uk/guidance/how-to-publish-on-gov-uk/images-and-videos'>read the image guidance</a> on using unique file names."),
```

Comment:
This is a nit-pick, but it's usually preferable to use double-quotes for HTML attributes.

```suggestion
    hint: raw('Images can be JPEG, PNG, SVG or GIF files. If you are uploading more than one image, <a class="govuk-link" href="https://www.gov.uk/guidance/how-to-publish-on-gov-uk/images-and-videos">read the image guidance</a> on using unique file names.'),
```